### PR TITLE
create[data]: add tokio3012.json

### PR DESCRIPTION
**Pull request by Tokio3012**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1550"></a>

### CODE REVIEW

**Review by Kartik2220 — APPROVED** [review]
nice code looks good

**Comment by Tokio3012** [comment]
@Kartik2220 😁 thanks!

File: public/data/vedanshi.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/tokio3012",
+      "icon": "twitter"
+    },
+    {
+      "name": "connect with me in discord",
+      "url": "https://discordapp.com/tokio/7761/",
+      "icon": "discord"
+    },
```
Comment (Panquesito7) [inline public/data/vedanshi.json:21]: There's a trailing comma in here.
```suggestion
    }
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/vedanshi.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/tokio3012",
+      "icon": "twitter"
+    },
+    {
+      "name": "connect with me in discord",
+      "url": "https://discordapp.com/tokio/7761/",
```
Comment (kumarsonsoff3) [inline public/data/vedanshi.json:19]: There is a broken link, please update it with the correct one! 🛠

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Few Fixes Required! 

- Update the filename with your `username.json` which should be `Tokio3012.json`
- Update your discord link with the correct one! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**